wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -1,4 +1,6 @@
 {% load wagtailadmin_tags i18n %}
+{% fragment as nav_icon_classes %}w-w-4 w-h-4{% endfragment %}
+{% fragment as nav_icon_button_classes %}w-w-slim-header w-h-slim-header w-bg-transparent w-border-transparent w-border-y-2 w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-text-meta w-transition hover:w-transform hover:w-scale-110 hover:w-text-text-label focus:w-text-text-label expanded:w-text-text-label expanded:w-border-b-current hover:expanded:w-border-b-transparent{% endfragment %}
```

Comment:
Before-after diff for ref – I’ve added `w-w-slim-header w-border-transparent w-border-y-2 expanded:w-border-b-current hover:expanded:w-border-b-transparent`, and also `expanded:w-text-text-label` is now set all the time.

Full diff:

```diff
- w-h-slim-header w-bg-transparent w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-text-meta w-transition hover:w-transform hover:w-scale-110 hover:w-text-text-label focus:w-text-text-label
+ w-w-slim-header w-h-slim-header w-bg-transparent w-border-transparent w-border-y-2 w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-text-meta w-transition hover:w-transform hover:w-scale-110 hover:w-text-text-label focus:w-text-text-label expanded:w-text-text-label expanded:w-border-b-current hover:expanded:w-border-b-transparent
```